src/main/resources/SLING-INF/nodetypes/nodetypes.cnd

```diff
@@ -31,3 +31,8 @@
 [pant:module] > nt:unstructured
 	- sling:resourceType (string) = 'pantheon/modules' mandatory autocreated
     + cachedContent (nt:unstructured) = nt:unstructured mandatory autocreated
+
+// Base type for titles
+[pant:title] > nt:unstructured
```

Comment:
Please look at my most recent changes for `pant:module` in the master branch and reflect them here for `pant:title` also.